overrides.nix

```diff
@@ -0,0 +1,112 @@
+pactSrc: pkgs: self: super: with pkgs.haskell.lib;
+let # Working on getting this function upstreamed into nixpkgs, but
+    # this actually gets things directly from hackage and doesn't
+    # depend on the state of nixpkgs.  Should allow us to have fewer
+    # github overrides.
+    callHackageDirect = {pkg, ver, sha256}@args:
+      let pkgver = "${pkg}-${ver}";
+      in self.callCabal2nix pkg (pkgs.fetchzip {
+           url = "http://hackage.haskell.org/package/${pkgver}/${pkgver}.tar.gz";
+           inherit sha256;
+         }) {};
+
+    ourOverrides = {
+      pact = dontCheck ( addBuildDepend (self.callCabal2nix "pact" pactSrc {}) pkgs.z3);
+
+      aeson = callHackageDirect {
+        pkg = "aeson";
+        ver = "1.4.3.0";
+        sha256 = "13lim8vv78m9lhn7qfjswg7ax825gn0v75gcb80hckxawgk8zxc1";
+      };
```

Comment:
This should be safe to unpin by now.

Reply:
It's not safe to unpin.  This nixpkgs comes with aeson-1.4.2.  :(

Reply:
Ah :(